services/utils/scheduler.go

```diff
@@ -1,4 +1,4 @@
-package activity
+package utils
```

Comment:
I was wondering if async isn't a better package for this.

Reply:
makes sense!